cmd/bsky-webhook/main.go

```diff
@@ -257,6 +257,11 @@ func readJetstreamMessage(ctx context.Context, jetstreamMessageEncoded []byte, b
 				return
 			}
 
+			// ignore users that are muted by the bluesky account running the service
+			if profile.Viewer.Muted {
+				return
```

Comment:
Is it worth logging that we skipped this one, for debugging purposes?

Reply:
Added